test/index.js

```diff
@@ -19,7 +19,7 @@ suite('nv', () => {
     assert.strictEqual(versions[0].start.toISOString(), '2018-04-24T00:00:00.000Z')
     assert.strictEqual(versions[0].lts.toISOString(), '2018-10-30T00:00:00.000Z')
     assert.strictEqual(versions[0].maintenance.toISOString(), '2020-04-01T00:00:00.000Z')
-    assert.strictEqual(versions[0].end.toISOString(), '2021-04-01T00:00:00.000Z')
+    assert.strictEqual(versions[0].end.toISOString(), '2021-04-30T00:00:00.000Z')
```

Comment:
Also, can you just revert these test changes?  The date and codenames were correct for that point in time, so I would prefer they do not update unless the the `schedule.json` changes and breaks them.